src/components/molecules/Visualizer/Engine/Cesium/useEngineRef.ts

```diff
@@ -74,6 +74,19 @@ export default function useEngineRef(
           options,
         );
       },
+      getViewport: () => {
+        const viewer = cesium.current?.cesiumElement;
+        if (!viewer || viewer.isDestroyed()) return;
+        const rect = viewer.camera.computeViewRectangle();
+        return rect
+          ? {
+              north: rect.north,
+              south: rect.south,
+              west: rect.west,
+              east: rect.east,
```

Comment:
```suggestion
              north: CesiumMath.toDegrees(rect.north),
              south: CesiumMath.toDegrees(rect.south,
              west: CesiumMath.toDegrees(rect.west),
              east: CesiumMath.toDegrees(rect.east),
```

Note: CesiumMath is imported with `import { Math as CesiumMath } from "cesium";`